src/types/pathways-model.d.ts

```diff
@@ -52,9 +55,9 @@ declare module 'pathways-model' {
   }
 
   export interface PathwayContextInterface {
```

Comment:
Is it better to have this in the `PathwayProvider`? It seems the other providers that we have define the context interface in the file.

Reply:
Welp, I tried moving it and Typescript was not happy so I reverted it back. It makes no sense to me, but if you can figure out how to move it without Typescript making a stink, let me know and I'll move it. ;)

Reply:
Wow very weird...

Seems you have to give the parameters a name
```
  interface PathwayContextInterface {
    pathways: Pathway[];
    status: string;
    addPathway: (pathway: Pathway) => void;
    deletePathway: (pathway: Pathway) => void;
  }
```

I have no idea why Typescript doesn't complain about it in the definitions file.

Reply:
Fixed!